test/helpers/nock_helper.js

```diff
@@ -2,7 +2,6 @@ const nock = require('nock')
 
 class NockHelper {
   mockRateLimit() {
-    nock.disableNetConnect()
```

Comment:
I see that `mockEndpoint` calls `mockRateLimit`. Do any tests rely on just `mockRateLimit`? If there are tests that call `mockRateLimit`, should we keep `disableNetConnect` in this function?